ironfish-cli/src/commands/chain/download.ts

```diff
@@ -66,17 +68,31 @@ export default class Download extends IronfishCommand {
     const node = await this.sdk.node()
     await NodeUtils.waitForOpen(node)
 
+    const networkId = node.internal.get('networkId')
+
+    let manifestUrl
+    if (flags.manifestUrl) {
+      manifestUrl = flags.manifestUrl
+    } else {
+      if (networkId === 0) {
+        // testnet
+        manifestUrl = Download.defaultTestnetManifestUrl
+      } else {
+        manifestUrl = Download.defaultMainnetManifestUrl
+      }
```

Comment:
Could we exit with an error if `networkId` is not 0 or 1 (Testnet or Mainnet), and `flags.manifestUrl` is not set? Something like:

```suggestion
      } else if (networkId === 1) {
        manifestUrl = Download.defaultMainnetManifestUrl
      } else {
        this.log(`Snapshots are not available for network ID ${networkId}`)
      }
```